client/src/diff/TextDiff.js

```diff
@@ -289,6 +299,7 @@ export default class TextDiffApp extends React.Component {
   constructor(props) {
     super(props);
     const colors = d3.scaleOrdinal().range([
+      window.infobase_color_constants.primaryColor,
       window.infobase_color_constants.primaryColor,
       window.infobase_color_constants.warnDarkColor,
       window.infobase_color_constants.successDarkColor,
```

Comment:
This doesn't make a huge amount of sense as a colour scale. Ideally you would want `indicator_added` to always be `successDarkColor`, `indicator_removed` to be `failDarkColor`, etc.

Suggest changing this to an object with keys the same as the object above.

Maybe define the keys once and then use `_.zipObject` to generate the object?